tasks/task04/Collect_tree_leaves.py

```diff
@@ -0,0 +1,36 @@
+def collect_leaves(tree, result=None):
+    if result is None: 
+        result = []
+    if(isinstance(tree, list)):
+        for i in tree:
+            if(isinstance(i, int)):
+                result.append(i)
```

Comment:
I would say, it's a bit over complicated here. for me if the item is a `list` you just have to `extend` the result with an item. check `list.extend` method